File-Formats/DocIO/Working-with-Word-document.md

```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-document/Apply-picture-background-to-document).
 
+## Hide background in print layout view
```

Comment:
## Hide background in the print layout view

Reply:
Done